chapters/zh-CN/_toctree.yml

```diff
@@ -0,0 +1,163 @@
+- title: 第0单元：欢迎来到Hugging Face音频课程！
+  sections:
+  - local: chapter0/introduction
+    title: 课堂目标
+  - local: chapter0/get_ready
+    title: 准备工作
+  - local: chapter0/community
+    title: 加入社区
+
+- title: 第1单元：音频数据处理
+  sections:
+  - local: chapter1/introduction
+    title: 单元简介
+  - local: chapter1/audio_data
+    title: 音频数据处理入门
+  - local: chapter1/load_and_explore
+    title: 加载音频数据集
+  - local: chapter1/preprocessing
+    title: 音频数据集的预处理
+  - local: chapter1/streaming
+    title: 音频数据的流式加载
+  - local: chapter1/quiz
+    title: 习题
+    quiz: 1
+  - local: chapter1/supplemental_reading
+    title: 补充阅读
+
+- title: 第2单元：音频应用的入门介绍
+  sections:
+  - local: chapter2/introduction
+    title: 音频应用概览
+  - local: chapter2/audio_classification_pipeline
+    title: 利用pipeline进行音频分类
+  - local: chapter2/asr_pipeline
+    title: 利用pipeline进行自动语音识别
+  - local: chapter2/hands_on
+    title: 实战练习
+
+- title: 第3单元：音频Transformer结构
+  sections:
+  - local: chapter3/introduction
+    title: Transformer模型回顾
+  - local: chapter3/ctc
+    title: CTC结构
+  - local: chapter3/seq2seq
+    title: Seq2Seq结构
+  - local: chapter3/classification
+    title: 音频分类结构
+  - local: chapter3/quiz
+    title: 习题
+    quiz: 3
+  - local: chapter3/supplemental_reading
+    title: 补充阅读
+
+- title: 第4单元：构建音乐风格分类器
```

Comment:
Please comment Chapter 4 in the TOC. As this chapter has not been translated yet, and there are no corresponding files, this breaks the doc-build.

Reply:
Commented U4 and events part in TOC!